src/components/molecules/OakTooltip/OakTooltip.tsx

```diff
@@ -27,22 +71,111 @@ export const OakTooltip = ({
         return "$btlr";
     }
   })();
-
   const borderRadiusProps: Partial<InternalTooltipProps> = {
     $borderRadius: "border-radius-m",
     [squaredCornerRadiusProp]: "border-radius-square",
   };
 
+  useLayoutEffect(() => {
+    if (!targetElement) {
+      return;
+    }
+
+    const updateOverlayStyle = () => {
+      const rect = targetElement.getBoundingClientRect();
+
+      setOverlayStyle({
+        top: rect.top,
+        left: rect.left,
+        width: rect.width,
+        height: rect.height,
+      });
+    };
+
+    const observer = new ResizeObserver(updateOverlayStyle);
+    observer.observe(targetElement);
+    observer.observe(document.documentElement);
+
+    return () => {
+      observer.disconnect();
+    };
+  }, [targetElement]);
+
   return (
-    <InternalTooltip
-      $background="bg-decorative5-main"
-      $color="text-primary"
-      $pv="inner-padding-m"
-      $ph="inner-padding-xl"
-      $font="heading-light-7"
-      tooltipPosition={tooltipPosition}
-      {...props}
-      {...borderRadiusProps}
-    />
+    <>
+      {createPortal(
+        isOpen && (
+          <OakBox
+            $position="absolute"
+            style={overlayStyle}
+            $pointerEvents="none"
+            $zIndex="modal-dialog"
+          >
+            <OakBox
+              $width="fit-content"
+              $height="fit-content"
+              $position="absolute"
+              {...getTooltipPositionProps(tooltipPosition)}
+            >
+              <InternalTooltip
+                $background="bg-decorative5-main"
+                $color="text-primary"
+                $pv="inner-padding-m"
+                $ph="inner-padding-xl"
+                $font="heading-light-7"
+                tooltipPosition={tooltipPosition}
+                {...props}
+                {...borderRadiusProps}
+              >
+                {tooltip}
+              </InternalTooltip>
+            </OakBox>
+          </OakBox>
+        ),
+        domContainer,
+      )}
+      <div
+        ref={(domElement) => {
+          setTargetElement(domElement?.firstElementChild ?? null);
+        }}
+        style={{ display: "contents" }}
```

Comment:
`display: contents` ensures that the element is ignored by the page for layout purposes (so it won't interfere with flex or grid) — it is only necessary as a way to get a reference to the target `Element` from within `children`.